runhouse/resources/hardware/cluster.py

```diff
@@ -862,6 +865,13 @@ def status(self, resource_address: str = None, send_to_den: bool = False):
             elif den_resp_status_code != 200:
                 logger.warning("Failed to send cluster status to Den")
 
+        if not configs.observability_enabled and status.get("env_servlet_processes"):
+            logger.warning(
+                "Observability not currently enabled on the cluster. Metrics are stale and will "
```

Comment:
@jlewitt1 Maybe "Observability currently not enabled on the cluster" or "Currently, observability is not enabled on the cluster"?